qiskit_experiments/library/characterization/ramsey_xy.py

```diff
@@ -208,3 +208,278 @@ def _metadata(self):
             if hasattr(self.run_options, run_opt):
                 metadata[run_opt] = getattr(self.run_options, run_opt)
         return metadata
+
+
+class StarkRamseyXY(BaseExperiment):
+    """A sign-sensitive experiment to measure the frequency of a qubit under pulsed Stark tone.
+
+    # section: overview
```

Comment:
Should the description link to the manual somewhere (I know it is linked in the options)?

Reply:
Ah thanks! I forgot to add the link.